tiup/tiup-cluster-no-sudo-mode.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Deploy and Maintain an Online TiDB Cluster Using TiUP No-sudo Mode
+summary: Learns how to deploy and maintain an online TiDB cluster using TiUP no-sudo mode.
+---
+
+# Deploy and Maintain an Online TiDB Cluster Using TiUP No-sudo Mode
+
+This document describes how to use the TiUP no-sudo mode to deploy a cluster.
+
+> **Note:**
+>
+> For CentOS, only CentOS 8 or later versions are supported.
+
+## Prepare the user and configure the SSH mutual trust
+
+1. Log in to all deployment target machines in sequence, and use the `root` user to create a user named `tidb` with the following command. In no-sudo mode, there is no need to configure password-free sudo for the `tidb` user, that is, you do not need to add the `tidb` user to sudoers.
+   
+    ```shell
+    adduser tidb
+    ```
+   
+2. Start the `systemd` user mode for the `tidb` user on every deployment target machine. This step is required and do not skip it.
+
+    1. Use the `tidb` user to set the `XDG_RUNTIME_DIR` environment variable.
+   
+        ```shell
+        mkdir -p ~/.bashrc.d
+        echo "export XDG_RUNTIME_DIR=/run/user/$(id -u)" > ~/.bashrc.d/systemd
+        source ~/.bashrc.d/systemd
+        ```
+   
+    2. Use `root` user to start user service.
+   
+        ```shell
+        $ systemctl start user@1000.service # `1000` is the ID of the tidb user. You can get the user ID by executing the `id` command.
+        $ systemctl status user@1000.service
+        user@1000.service - User Manager for UID 1000
+        Loaded: loaded (/usr/lib/systemd/system/user@.service; static; vendor preset>
+        Active: active (running) since Mon 2024-01-29 03:30:51 EST; 1min 7s ago
+        Main PID: 3328 (systemd)
+        Status: "Startup finished in 420ms."
+        Tasks: 6
+        Memory: 6.1M
+        CGroup: /user.slice/user-1000.slice/user@1000.service
+                ├─dbus.service
+                │ └─3442 /usr/bin/dbus-daemon --session --address=systemd: --nofork >
+                ├─init.scope
+                │ ├─3328 /usr/lib/systemd/systemd --user
+                │ └─3335 (sd-pam)
+                └─pulseaudio.service
+                  └─3358 /usr/bin/pulseaudio --daemonize=no --log-target=journal
+        ```
+   
+    Execute `systemctl --user` in the terminal. If no errors are reported, it indicates that it has started normally.
+
+3. Use ssh-keygen in the central control computer to generate a key and copy the public key to other deployment machines.
```

Comment:
Compared with the Chinese version, the following is missed. Please check whether this is on purpose.

```
使用 `root` 用户执行以下命令，为 systemd 用户 `tidb` 启用驻留。

    ```bash
    loginctl enable-linger tidb
    ```
    更多详情，参见 [systemd 用户实例的自动启动](https://wiki.archlinux.org/title/Systemd/User#Automatic_start-up_of_systemd_user_instances)。
```